src/assets/i18n/fr.json

```diff
@@ -86,6 +86,7 @@
           }
         },
         "trainrunsection-validator": {
+          "broken-symmetry": "Symétrie brisée",
           "target-arrival-not-reacheable": {
             "title": "Avertissement d'arrivée à la destination",
```

Comment:
Instead
... arrivée à la destination
I suggest 
"Avertissement d'arrivée à destination", 
(without article "la")
qu'en pensez vous @emersion & @louisgreiner ?

Reply:
@louisgreiner - please fix and make a suggestion commit :-)